opdracht5.js

```diff
@@ -0,0 +1,304 @@
+//     zoals `inventory[0].availableSizes`) en de groottes op de volgende manier
+// samenvoegt: `[schermgrootte] inches ([schermgrootte omgerekend]cm) | [schermgrootte] inches ([schermgrootte omgerekend]cm)`
+// etc. Als een tv maar één schermgrootte heeft (`[32]`) wordt de output `32 inch (81 cm)`. Wanneer een tv vier
+// schermgroottes heeft (`[43, 50, 55, 58]`) wordt de output `43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)`.
+//     Test of jouw functie werkt in alle mogelijke gevallen.
+
+
+function testArray (arrayInvent) {
+    let inchToCm = ""
+    for (let i = 0; i < arrayInvent.length ; i++) {
+        const tvSizeInch = arrayInvent[i]
+        const tvSizeCm = tvSizeInch * 2.54
+        inchToCm = inchToCm + `${tvSizeInch} Inch | ${tvSizeCm}CM | `
+    } return inchToCm
+}
+console.log(testArray(inventory[0].availableSizes))
+
+
+// * **Opdracht 5d:** Zorg ervoor de informatie van één van de tv's zoals het voorbeeld wordt weergegeven op de pagina.
+// Gebruik hiervoor de functies die je hebt gemaakt in opdracht 5a, 5b en 5c.
+// verzamel alle functies onder 1 functie
+//   ```
+//   Philips 43PUS6504/12 - 4K TV
+//   €379,-
+//   43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)
+//
+//   ```
+
+const tvSelection = document.getElementById("tv-selection")
+tvSelection.setAttribute("class", "css-div-tv")
+
+const divTv = document.createElement("li")
+const divTv1 =document.createElement("li")
+const divTv2 = document.createElement("li")
+
+divTv.textContent = testArray(inventory[0].availableSizes)
+divTv1.textContent = arrayPriceTv(inventory)[0]
+divTv2.textContent = printTV(inventory)[0]
+tvSelection.appendChild(divTv2)
+tvSelection.appendChild(divTv1)
+tvSelection.appendChild(divTv)
+
+// * **Opdracht 5e:** Schrijf een functie die ALLE tv's weergeeft op de pagina zoals in het voorbeeld. Dit wil je
+// natuurlijk niet acht keer opnieuw schrijven, want nu zijn het 8 tv's, maar in de toekomst misschien wel 200!
+// Gebruik in deze functie de voorgaande functies die je hebt geschreven, om onderdelen van de data te formatten. De
+// overkoepelende "tv-generator-functie" verwacht één parameter: de volledige array met tv-objecten. Vergeet 'm niet aan
+// te roepen!
+
+
+function tvGenFunction () {
+    const tvSelection = document.getElementById("tv-functie")
+    tvSelection.setAttribute("class", "css-div-function")
+
+    const divTv = document.createElement("li")
+    const divTv1 =document.createElement("li")
+    const divTv2 = document.createElement("li")
+
+    divTv.textContent = testArray(inventory[0].availableSizes)
+    divTv1.textContent = arrayPriceTv(inventory)[0]
+    divTv2.textContent = printTV(inventory)[0]
+    tvSelection.appendChild(divTv2)
+    tvSelection.appendChild(divTv1)
+    tvSelection.appendChild(divTv)
+}
+tvGenFunction()
+
+// Dit had ik eerst gemaakt, al had ik snel door dat ik de elementen niet kon aanspreken voor styling.
```

Comment:
Foei, dat is niet waar ;-) :-D want: zélfs al zou je meerdere HTML elementen hebben met allemaal dezelfde Type en dezelfde Class, dan kun je deze nog steeds individueel stylen! Alleen moet je dan zeker weten hoeveel precies je van elk element hebt = dus nieet dynamisch. Maar als je bijvoorbeeld 8 elementen hebt dan kun je ze stylen d.m.v. de CSS selector 'n' of met 'child' - dus bijvoorbeeld zo: `DIV:nth-child(1) {}, DIV:nth-child(2) {}, ` - zie ook: https://www.w3schools.com/cssref/sel_nth-child.asp